model/src/main/java/userinfo/ClientInfo.java

```diff
@@ -0,0 +1,44 @@
+package userinfo;
+
+/**
+ * Stores the information of a client.
+ */
+public class ClientInfo extends UserInfo {
+    private String firstName;
+    private String lastName;
+
+    /**
+     * Constructs a ClientInfo object.
+     *
+     * @param address: the address of this client
+     * @param phoneNumber: the phone number of this client
+     * @param firstName: the first name of this client
+     * @param lastName: the last name of this client
+     */
+    public ClientInfo(String address, String phoneNumber, String firstName, String lastName) {
+        super(address, phoneNumber);
+        this.firstName = firstName;
+        this.lastName = lastName;
+    }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public void setFirstName(String firstName) {
+        this.firstName = firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
+
+    public void setLastName(String lastName) {
+        this.lastName = lastName;
+    }
```

Comment:
These should be documented, it doesn't have to be super detailed, for getters, generally you only need to fill in the @returns.